rakelib/plugin.rake

```diff
@@ -23,7 +23,9 @@ namespace "plugin" do
 
   task "install-all" do
     Rake::Task["vendor:bundle"].invoke("tools/Gemfile.plugins.all")
+    if ENV["LS_ENV"] == "test"
```

Comment:
in vendor.rake below you use `LogStash::Environment` but not here? why not `LogStash::Environment.test?` ?